operator-framework-core/src/main/java/io/javaoperatorsdk/operator/AggregatedOperatorException.java

```diff
@@ -1,23 +1,30 @@
 package io.javaoperatorsdk.operator;
 
 import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
 
 public class AggregatedOperatorException extends OperatorException {
+
   private final List<Exception> causes;
 
   public AggregatedOperatorException(String message, Exception... exceptions) {
-    super(message);
-    this.causes = exceptions != null ? Arrays.asList(exceptions) : Collections.emptyList();
+    super(message, exceptions[0]);
```

Comment:
This might NPE if no exception is passed.

Reply:
pushed with null check